.travis.yml

```diff
@@ -1,16 +1,34 @@
 language: rust
+
+before-install:
+  - curl --location https://github.com/mozilla/grcov/releases/latest/download/grcov-linux-x86_64.tar.bz2 | tar jxf -
```

Comment:
I am still not sure why this is in `before-install`

Reply:
I will move this to be a step in the script

Reply:
I think that was actually somehow messing up what percentage was reported as the full coverage. 🤔  no idea to be honest, glad that now we get a % that makes sense.

Reply:
yeah, sorry didn't see that they did that in the grcov example there ... not sure why they would do that 🤔 but yeah if it works now :P